README.md

```diff
@@ -1,7 +1,41 @@
 # CLA Public
 
+## Important notices
+
+- `master` branch is the default branch
```

Comment:
`master` as the default branch seems to be the standard practice across GitHub repositories. Should we mention it?